_posts/2023-12-07-semantic-options-benchmarks.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Semantic Search with OpenSearch: Architecture options and Benchmarks"
+authors:
+ - sean-zheng-amazon
```

Comment:
None of the authors are appearing. Are there entries in _authors with a matching 'short_name' ?